classes/task/FileLoader.inc.php

```diff
@@ -387,27 +387,6 @@ private function _stageFile()
     {
         $this->moveFile($this->_processingPath, $this->_stagePath, $this->_claimedFilename);
     }
-
-    /**
-     * Send the passed message to the administrator by email.
-     *
-     * @param string $message
-     */
-    private function _notify($message, $messageType)
```

Comment:
Unused method.

Besides the `new Mail()`, there are just other two places using the old Mail class. I was going to update, but I think Vitaliy is working on it.

Reply:
Yes, tagging @Vitaliy-1.

Reply:
Thanks, I'll replace those with correspondent Mailable classes